test.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 # Running Parity Full Test Sute
 
-FEATURES="--features json-tests"
+FEATURES="--features json-tests ipc"
```

Comment:
I'm pretty sure it won't work:
`cargo test test_name` - runs only tests with `test_name`
`cargo test --features json-tests ipc` - runs only tests with `ipc`